mock-caas/src/main/java/com/sequenceiq/caas/grpc/service/auth/MockUserManagementService.java

```diff
@@ -531,6 +533,7 @@ public void getAccount(GetAccountRequest request, StreamObserver<GetAccountRespo
                                 .setWorkloadSubdomain(ACCOUNT_SUBDOMAIN)
                                 .addEntitlements(createEntitlement(CDP_AZURE))
                                 .addEntitlements(createEntitlement(CDP_AUTOMATIC_USERSYNC_POLLER))
+                                .addEntitlements(createEntitlement(CDP_CLOUD_IDENTITY_MAPPING))
```

Comment:
I think we should have an env var to enable and disable the entitlement. See a couple of lines above.